classes/local/step/connector_curl.php

```diff
@@ -213,24 +221,51 @@ public function execute($input = null) {
 
         $this->enginestep->log($config->curl);
 
-        $dbgcommand = 'curl -X ' .  strtoupper($method) . ' ' . $config->curl;
+        // Construct a bash curl command.
+        // See https://manpages.org/curl.
+        $dbgcommand = 'curl -s -X ' .  strtoupper($method) . ' ' . $config->curl;
 
-        if (!empty($config->timeout)) {
-            $this->timeout = (int) $config->timeout;
-            $dbgcommand .= ' --max-time ' . $config->timeout;
+        // Extract timeout.
+        $timeout = (int) $config->timeout;
+        if (empty($timeout)) {
+            $timeout = self::DEFAULT_TIMEOUT;
```

Comment:
This may be a potential candidate for a ternary operator?
```suggestion
        $timeout = (int) $config->timeout ?: self::DEFAULT_TIMEOUT;
```